api/api.go

```diff
@@ -56,6 +57,7 @@ type API struct {
 	linuxUpgradeController      linuxupgrade.Controller
 	kubernetesInstallController kubernetesinstall.Controller
 	kubernetesUpgradeController kubernetesupgrade.Controller
+	migrationController         migration.Controller
```

Comment:
i think we need to prefix everything with `kurl` as we might have other types of migrations in the future